lib/vec.c

```diff
@@ -739,9 +742,33 @@ M0_INTERNAL bool m0_ivec_cursor_move_to(struct m0_ivec_cursor *cur,
 		if (ret)
 			break;
 	}
+
 	return ret;
 }
 
+M0_INTERNAL m0_bindex_t m0_ivec_cursor_conti(const struct m0_ivec_cursor *cur,
```

Comment:
My comment was accidentally deleted by me.
I will repost it here.
My question is: It is OK to have @dest less than current cursor postion?

@andriytk replied:

No, it's not OK. Similar to move_to(), we know nothing about the segments before our cursor position, so returning dest in this case would be a false positive result.

Besides, I'm not sure where this can be needed. It rather sounds like a bug in the code, that's why we put this check (that dest >= cursor_index) at PRE().